pulsar-broker/src/main/java/org/apache/pulsar/broker/ManagedLedgerClientFactory.java

```diff
@@ -93,7 +93,7 @@ public void initialize(ServiceConfiguration conf, MetadataStoreExtended metadata
                     try {
                         return bookkeeperProvider.create(conf, metadataStore, eventLoopGroup,
                                 Optional.ofNullable(ensemblePlacementPolicyConfig.getPolicyClass()),
-                                ensemblePlacementPolicyConfig.getProperties());
+                                ensemblePlacementPolicyConfig.getProperties(), statsLogger);
```

Comment:
It may introduce another problem here, duplicate metrics owned by different `OrderedExecutors`, cannot figure which thread if a metric goes wrong.

Reply:
@Shawyeok Thanks for your feedback. I generate bk client ordered thread pool metrics name by `ISOLATION_BOOKIE_GROUPS` name configured in `ensemblePlacementPolicyConfig`. So that different bk client has different metric name.